apps/src/sites/studio/pages/levels/_pixelation.js

```diff
@@ -1,4 +1,3 @@
-/* global $ appOptions dashboard options */
 // This file does not pass eslint.
```

Comment:
does this file now pass eslint? I wonder why we had this turned off in the first place rather than just linting it.

Reply:
Shows how laser focused I was on the `/* globals ... */` comments that I didn't even pay any attention to the fact that it was disabled there. 😆 

That file can pass linting now - it just needs a function deleted and strict equality checks. I'm gonna toss it into a separate PR for tidiness' sake, to ensure there's no functionality changes in this one.